connectivity/drivers/mbedtls/FEATURE_CRYPTOCELL310/CMakeLists.txt

```diff
@@ -8,8 +8,7 @@ target_include_directories(mbed-os
     PUBLIC
         ${CMAKE_CURRENT_SOURCE_DIR}
         ${CMAKE_CURRENT_SOURCE_DIR}/include
-        ${CMAKE_CURRENT_SOURCE_DIR}/include/cryptocell310
-        ${CMAKE_CURRENT_SOURCE_DIR}/include/internal
+        ${CMAKE_CURRENT_SOURCE_DIR}/include/cryptocell310/internal
```

Comment:
I have seen header files included with paths starting a number of directories before the file is. To make sure all is covered, I include all directories.
So I would have:
```suggestion
        ${CMAKE_CURRENT_SOURCE_DIR}/include/cryptocell310
        ${CMAKE_CURRENT_SOURCE_DIR}/include/cryptocell310/internal
```

Reply:
Correct, fixing now, also internal should be private

Reply:
There are a number of dirs that should be private. However, I thought we should do it once we start breaking down the library into small components.

Reply:
Yes, we will need to clean it. I put it back as it failed, investigating